Notre Dame Philosophical Reviews.js

```diff
@@ -0,0 +1,207 @@
+{
+  "translatorID": "a823550b-6475-4b20-8539-a3c416906228",
+  "label": "Notre Dame Philosophical Reviews",
+  "creator": "Emiliano Heyns",
+  "target": "^https://ndpr\\.nd\\.edu",
+  "minVersion": "2.1",
+  "maxVersion": "",
+  "priority": 100,
+  "inRepository": true,
+  "translatorType": 4,
+  "browserSupport": "gcsibv",
+  "lastUpdated": "2016-01-04 20:13:01"
+}
+
+// Generated by CoffeeScript 1.10.0
+
+/*
+Notre Dame Philosophical Reviews Translator
+Copyright (C) 2016 Emiliano Heyns
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+var detectWeb, doWeb, scrape, testCases;
+
+detectWeb = function(doc, url) {
+  if (url.match(/^https:\/\/ndpr\.nd\.edu\/news\/[0-9]+/)) {
```

Comment:
for literal matches like the two below, we prefer `url.indexOf("string")!=-1`; where you do need regex, such as here, use `url.search(/regex/)!=-1` both for efficiency. Since detectWeb runs a fair amount, we care about even pretty small efficiency gains.